packages/core/src/middleware/koa-error-handler.ts

```diff
@@ -24,6 +24,11 @@ export default function koaErrorHandler<StateT, ContextT, BodyT>(): Middleware<
         return;
       }
 
+      // Koa will handle `HttpError` with a built-in manner.
+      if (error instanceof HttpError) {
+        return;
+      }
```

Comment:
Wondering what is the expected behavior after the updates, is there any specific use case?